scripts/load-register-tsv.sh

```diff
@@ -9,7 +9,7 @@ source "$OPENREGISTER_BASE/deployment/scripts/includes/set-vars.sh"
 
 usage()
 {
-  echo "usage: ./load-register-tsv.sh [register] [phase] [tsvfile relative to root] [local|remote] [custodian] [optional data dir relative to root if not REGISTER-data]"
+  echo "usage: ./load-register-tsv.sh [register] [phase] [tsvfile relative to root] [metadata|user|all] [local|remote] [custodian] [optional data dir relative to root if not REGISTER-data]"
```

Comment:
I'm wondering whether it's worth calling this `system` instead of metadata, just to be consistent with the entry type?

Reply:
Sure.